contracts/proposal/SlashingRefundProposal.sol

```diff
@@ -4,13 +4,7 @@ pragma solidity 0.8.27;
 import {DelegatecallExecutableProposal} from "./base/DelegatecallExecutableProposal.sol";
 import {Cancelable} from "./base/Cancelable.sol";
 import {Strings} from "@openzeppelin/contracts/utils/Strings.sol";
-
-/// @notice An interface to update slashing penalty ratio
-interface SFC {
-    function updateSlashingRefundRatio(uint256 validatorID, uint256 ratio) external;
-
-    function isSlashed(uint256 validatorID) external returns(bool);
-}
+import {SFC} from "../adapters/SFCToGovernable.sol";
```

Comment:
Please move the SFC interface into a standalone file. Into directory like `contracts/interfaces`
We don't want the SFCToGovernable adapter dependency in a proposal implementations.